lib/ansible/modules/identity/ipa/ipa_dnsrecord.py

```diff
@@ -40,6 +42,7 @@
     required: false
     default: 'A'
     choices: ['A', 'AAAA', 'A6', 'CNAME', 'DNAME', 'PTR', 'TXT', 'SRV', 'MX']
```

Comment:
```suggestion
    choices: ['A', 'AAAA', 'A6', 'CNAME', 'DNAME', 'PTR', 'TXT', 'SRV', 'MX']
```
i'd put them alphabetically too